src/plugins/discover/public/application/components/context_error_message/context_error_message.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { EuiCallOut, EuiText } from '@elastic/eui';
+import { FormattedMessage, I18nProvider } from '@kbn/i18n/react';
+// @ts-ignore
+import { FAILURE_REASONS, LOADING_STATUS } from '../../angular/context/query';
+
+export interface ContextErrorMessageProps {
+  /**
+   * the status of the loading action
+   */
+  status: string;
+  /**
+   * the reason of the error
+   */
+  reason?: string;
+  /**
+   * parameters used for invalid tieBreakerFields realted errors
+   */
+  queryParameters: { indexPatternId: string };
+}
+
+export function ContextErrorMessage({ status, reason, queryParameters }: ContextErrorMessageProps) {
+  if (status !== LOADING_STATUS.FAILED) {
+    return null;
+  }
+  return (
+    <I18nProvider>
+      <EuiCallOut
+        title={
+          <FormattedMessage
+            id="discover.context.failedToLoadAnchorDocumentDescription"
+            defaultMessage="Failed to load the anchor document"
+          />
+        }
+        color="danger"
+        iconType="alert"
+        data-test-subj="contextErrorMessageTitle"
+      >
+        <EuiText data-test-subj="contextErrorMessageBody">
+          {reason === FAILURE_REASONS.INVALID_TIEBREAKER && (
+            <FormattedMessage
+              id="discover.context.noSearchableTiebreakerFieldDescription"
+              defaultMessage="No searchable tiebreaker field could be found in the index pattern {indexPatternId}. Please change the advanced setting {tieBreakerFields} to include a valid field for this index pattern."
+              values={{
+                indexPatternId: queryParameters.indexPatternId,
+                tieBreakerFields: <code>context:tieBreakerFields</code>,
+              }}
+            />
+          )}
```

Comment:
I think this error message is redundant, legacy, might have been useful in recent time. I tried to set the `tieBreakerFields` to an invalid value, but I that case `_doc` is used as tiebreaker. And I think this is fine this way. So unless you find a way to break this to have this message displayed, I'd suggest to remove it.

Reply:
I tried some ideas but didn't work. I'll remove it.